custom_components/tuya_local/devices/ezaiot_r9lite_thermostat.yaml

```diff
@@ -0,0 +1,219 @@
+      type: integer
+      name: unknown_101
+    - id: 112
+      type: integer
+      name: unknown_112
+    - id: 102
+      type: integer
+      name: Temperature adjust step
+    - id: 110
+      type: boolean
+      name: Valve state boolean
+    - id: 105
+      type: string
+      name: Device name
+    - id: 108
+      type: integer
+      name: Temperature calibration step
+secondary_entities:
+  - entity: switch
+    translation_key: anti_frost
+    category: config
+    dps:
+      - id: 10
+        type: boolean
+        name: switch
+  - entity: select
+    translation_key: temperature_unit
+    category: config
+    dps:
+      - id: 23
+        type: string
+        name: option
+        mapping:
+          - dps_val: c
+            value: celsius
+          - dps_val: f
+            value: fahrenheit
+  - entity: select
+    name: Schedule
+    category: config
+    icon: 'mdi:calendar-clock'
+    dps:
+      - id: 31
+        type: string
+        name: option
+        mapping:
+          - dps_val: '5_2'
+            value: Weekday+Weekend
+          - dps_val: '6_1'
+            value: Mon-Sat+Sun
+          - dps_val: '7'
+            value: Daily
+  - entity: binary_sensor
+    name: Valve
+    class: opening
+    category: diagnostic
+    dps:
+      - id: 36
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: open
+            value: true
+          - dps_val: close
+            value: false
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 40
+        type: boolean
+        name: lock
+  - entity: binary_sensor
+    category: diagnostic
+    class: problem
+    dps:
+      - id: 45
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+      - id: 45
+        type: bitfield
+        name: fault_code
+  - entity: number
+    name: High temperature limit
+    category: config
+    class: temperature
+    dps:
+      - id: 19
+        type: integer
+        name: value
+        unit: C
+        range:
+          min: 50
+          max: 10000
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: number
+    name: Low temperature limit
+    category: config
+    class: temperature
+    dps:
+      - id: 26
+        name: value
+        type: integer
+        unit: C
+        range:
+          min: 50
+          max: 10000
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: number
+    name: Temperature calibration
+    category: config
+    class: temperature
+    dps:
+      - id: 109
+        name: value
+        type: integer
+        unit: C
+        range:
+          min: -99
+          max: 99
+        mapping:
+          - scale: 10
+            step: 1
+  - entity: sensor
+    #The dps id 113 is readonly (set not working)- I don't know why..probably not implemented on device
+    name: Temperature sensor type
```

Comment:
I don't think this needs to be a sensor. It can just be an attribute of the climate entity.

Reply:
Fixed. https://github.com/janouskovec/tuya-local/commit/3cf2d019f6d9b964a3a553a4271c566c746ec14f